docs/_config.yml

```diff
@@ -9,4 +9,4 @@ defaults:
       path: "" # an empty string here means all files in the project
     values:
       cdn_path: https://ir.ebaystatic.com/cr/v/c1/skin
-      version: 15.0.0
+      version: 15.1.0-1
```

Comment:
Does this need to get updated to `15.1.0` before the release or something changes it automatically after?